src/components/analytics/ExampleAnalyticsIntegration.tsx

```
@@ -0,0 +1,260 @@
+            onMouseEnter={() => handleTooltipOpen('max-button-tooltip')}
+          >
+            Max
+          </Button>
+        </HStack>
+      </Box>
+
+      {/* Action Buttons */}
+      <HStack spacing={4}>
+        <Button
+          colorScheme="blue"
+          onClick={handleApprove}
```

Biome (2.1.2)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)